include/zephyr/bluetooth/classic/spp.h

```diff
@@ -0,0 +1,137 @@
+/** @file
+ *  @brief Bluetooth SPP Protocol handling.
+ */
+
+/*
+ * Copyright 2025 Xiaomi Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SPP_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SPP_H_
+
+/**
+ * @brief SPP
+ * @defgroup SPP
+ * @ingroup bluetooth
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/bluetooth/buf.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/uuid.h>
+
+#include <zephyr/bluetooth/classic/rfcomm.h>
+
+struct bt_spp {
+	uint8_t connected: 1;

```

Comment:
Shouldn't this be `bool`? Seems a bit racy to expose it though, since the state may change between reading this value and calling e.g. `send()` (in which case `send()` would return an error, so why do you even need a `connected` then?

Reply:
Sorry, this is a mark left from the previous version. The current version has recorded it through acl_conn